source/clang/package.d

```diff
@@ -428,10 +451,6 @@ struct Cursor {
         return Cursor(clang_getSpecializedCursorTemplate(cx));
     }
 
-    TranslationUnit translationUnit() @safe nothrow const {
```

Comment:
Removed because `clang_Cursor_getTranslationUnit` can return null in case if called on "invalid" cursor